### PR TITLE
Hide unverified token search results for short queries

### PR DESCRIPTION
**Pull request by christianbaroni**
## What changed (plus any additional context for devs)

- Speeds up the swap output search by filtering out unverified results if the search query length is less than 3 characters

### CODE REVIEW

**Review by derHowie — APPROVED** [review]
👌

**Review by greg-schrammel — APPROVED** [review]
isn't it worth it disabling the unverified tokens query too?

**Comment by christianbaroni** [comment]
> isn't it worth it disabling the unverified tokens query too?

@greg-schrammel @derHowie pushed a change for this, mind taking another look?

File: src/entries/popup/hooks/useSearchCurrencyLists.ts [hunk 1]
```diff
@@ -167,6 +168,7 @@ export function useSearchCurrencyLists({
     threshold,
     query,
     fromChainId,
+    disabled: disableUnverifiedSearch,
   });
```
Comment (greg-schrammel) [inline src/entries/popup/hooks/useSearchCurrencyLists.ts:172]: I think useTokenSearch already accepts a config as second arg
```ts
  const {
    data: targetUnverifiedAssets,
    isLoading: targetUnverifiedAssetsLoading,
  } = useTokenSearch(
    {
      chainId: outputChainId,
      keys,
      list: 'highLiquidityAssets',
      threshold,
      query,
      fromChainId,
    },
    { enabled: query.trim().length < 3 },
  );
  ```

**Review by estebanmino — APPROVED** [review]

**Pull request merged**